lib/ui/index.js

```diff
@@ -1,21 +1,20 @@
 'use strict';
-const ora       = require('ora');
-const omit      = require('lodash/omit');
-const chalk     = require('chalk');
-const execa     = require('execa');
-const Listr     = require('listr');
-const Table     = require('cli-table3');
-const Promise   = require('bluebird');
-const inquirer  = require('inquirer');
-const isObject  = require('lodash/isObject');
-const stripAnsi = require('strip-ansi');
-const ListrError = require('listr/lib/listr-error');
-const logSymbols = require('log-symbols');
-const isFunction = require('lodash/isFunction');
-
-const errors = require('../errors');
+
+const ora         = require('ora');
+const Promise     = require('bluebird');
+const isObject    = require('lodash/isObject');
+const logSymbols  = require('log-symbols');
+const isFunction  = require('lodash/isFunction');
+const lazyRequire = require('import-lazy')(require);
 const CLIRenderer = require('./renderer');
 
+const omit = lazyRequire('lodash/omit');
+const chalk = lazyRequire('chalk');
```

Comment:
`log-symbols` requires chalk the normal way - is there any performance benefit by lazy loading it here?

Reply:
It's also fully required [here](https://github.com/TryGhost/Ghost-CLI/blob/1.8.1/lib/commands/ls.js#L6) 🤔